runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
I think I figured out a good use for `'highlight'` in unit testing:

```lua
function reset_highlight(except)
  local hl_flags = {
    '8', '~', 'z', 'Z', '@', 'd', 'e', 'i', 'l', 'm', 'M', 'n',
    'N', 'r', 's', 'S', 'c', 't', 'v', 'V', 'w', 'W', 'f', 'F',
    'A', 'C', 'D', 'T', '-', '>', 'B', 'P', 'R', 'L', '+', '=',
    'x', 'X', '*', '#', '_', '!', '.', 'o'
  }

  local hl = ''
  local found = false

  for i=0,#hl_flags do
    found = false
    for j=0,#except do
      if except[j] == hl_flags[i] then
        found = true
        break
      end
    end

    if not found then
      hl = hl .. hl_flags[i] .. '-,'
    end
  end

  meths.set_option('highlight', hl)
end

...

reset_highlight({'e', 's', 'S'})
```

Using that makes it so I don't have to setup a bunch of ignored highlights for the screen tests while retaining a few that I'm interested in seeing.